libs/ui/src/components/Explorer/TableCell.tsx

```diff
@@ -145,25 +146,49 @@ export const TableCell: FunctionComponent<ITableCellProps> = ({values, attribute
             });
             return <TableTagGroup tags={tags} />;
         } else if (isLinkValues(values, attributeProperties)) {
-            return (
-                <KitAvatar.Group max={{count: 5}}>
-                    {values.map((value, index) => {
-                        if (!isLinkValue(value, attributeProperties)) {
-                            return null;
-                        }
-
-                        return (
-                            <KitAvatar
-                                key={index}
-                                label={String(value?.linkPayload?.whoAmI.label)}
-                                src={value?.linkPayload?.whoAmI.preview?.small}
-                                color="primary"
-                                secondaryColorInvert
-                            />
-                        );
-                    })}
-                </KitAvatar.Group>
-            );
+            switch (attributeProperties.multi_link_display_option) {
+                case MultiLinkDisplayOption.avatar:
+                    return (
+                        <KitAvatar.Group max={{count: 5}}>
+                            {values.map((value, index) => {
+                                if (!isLinkValue(value, attributeProperties)) {
+                                    return null;
+                                }
+
+                                return (
+                                    <KitAvatar
+                                        key={index}
+                                        label={String(value?.linkPayload?.whoAmI.label)}
+                                        src={value?.linkPayload?.whoAmI.preview?.small}
```

Comment:
**nitpick:** C'est surement une erreur qui est passée avant mais si on vérifie que c'est bien une `ILinkValue` au dessus c'est que la `value.linkPayload` doit être définie. On doit pouvoir écrire :
```suggestion
                                        label={String(value.linkPayload.whoAmI.label)}
                                        src={value.linkPayload.whoAmI.preview?.small}
```
Et ce en dessous aussi

Reply:
ts n'est pas content si je l'enlève, vu que ça marche je suis d'avis de laisser en l'état , mieux trop que pas assé pour les check d'existence ?

Reply:
dingue c'est vraiment mal typer la génération ! faudrait changer des choses dans le back, bon c'est chaint mais tu as raison, si tu repasses sur la pr on peut quand même mettre 
```suggestion
                                        label={String(value.linkPayload?.whoAmI.label)}
                                        src={value.linkPayload?.whoAmI.preview?.small}
```
C'est au moins le minimum qui passe :)

Reply:
même chose sur le truc un peu en dessous, ligne 175